src/main/java/UserInput.java

```diff
@@ -0,0 +1,10 @@
+import java.util.Scanner;
+
+public class UserInput {
+    public static void main(String[] args) {
```

Comment:
Right now you can enter only 1 number and program will be finished.
I want the following scenario:
>>Please enter count of numbers you want to pass:
>>4
>>Enter number #1:
>>55
>>Enter number #2:
>>3
>>Enter number #3:
>>-8
>>Enter number #4:
>>62
>>You've entered: [55, 3, -8, 62]